docs/manual/installation.rst

```diff
@@ -1,23 +1,37 @@
 Installation
 ============
 
-**phpspec** is a php 5.3+ library that you'll have in your project
-development environment. Before you begin, ensure that you have at least
-PHP 5.3.3 installed.
+**phpspec** is a php 5.6+ library that you'll have in your project
+development environment. Before you begin, ensure that you have
+PHP 5.6 or 7 installed.
 
-Method #1 (Composer)
---------------------
+Installation process:
+---------------------
 
-The simplest way to install phpspec with all its dependencies is through
-Composer.
+You can install phpspec with all its dependencies through Composer. Follow
+instructions on `the composer website <https://getcomposer.org/download/>`_ if
+you don't have it installed yet.
 
-First, create a ``composer.json`` file in your project's root directory:
+Method #1 (Composer command):
+-----------------------------
+
+You can use this Composer command to install phpspec:
+
+.. code-block:: bash
+
+    composer require phpspec/phpspec
+
+Method #2 (Composer config file):
+---------------------------------
+
+If you prefer editing your ``composer.json`` file manually, add phpspec to your
+``require-dev`` section like this:
 
 .. code-block:: js
 
     {
         "require-dev": {
-            "phpspec/phpspec": "~2.0"
+            "phpspec/phpspec": "*"
```

Comment:
Star isn't a very good choice for a dependency in my opinion. The better choice would be to make it specific `^2.5` or use a pseudo text `insert-version-requirement`.

But in the end, in my opinion, it would actually be best to remove config file installation completely, too complicated and too much chances for messing something up.